mc2/controllers/docker/tests/test_docker_controller.py

```diff
@@ -224,7 +225,8 @@ class DockerControllerHypothesisTestCase(TestCase):
     """
 
     @hsettings(perform_health_check=False)
-    @given(_r=random_module(), controller=docker_controller())
+    @given(_r=random_module(),
+           controller=docker_controller(marathon_cmd=text(["ping"])))
```

Comment:
@smn: perhaps the thing we need to do here is validate the form data by making sure it's `shlex.split`-able. I'm not really sure how to write a Hypothesis text generator that will only produce valid shell command syntax, though.